### ISSUE TITLE
Stopped working with latest Firefox nightly.

### ISSUE THREAD

**Issue by Betsy25** [jtackaberry/nosquint#195]
Some code has changed in the [Latest nightly](http://forums.mozillazine.org/viewtopic.php?f=23&t=2987015) of Firefox, which broke Nosquint from functioning.

**Comment by streetwolf** [comment]
I don't think we will ever see another version of NS.  Try this as a replacement.  Almost as good but not quite IMO.  https://addons.mozilla.org/en-US/firefox/addon/zoom-page/
